source/event_stream_rpc_server.c

```diff
@@ -922,13 +922,6 @@ static void s_route_message_by_type(
                     AWS_LS_EVENT_STREAM_RPC_SERVER,
                     "id=%p: stream_id is invalid because it's not sequentially increasing",
                     (void *)connection);
-
-                aws_raise_error(AWS_ERROR_EVENT_STREAM_RPC_PROTOCOL_ERROR);
-                s_send_connection_level_error(
```

Comment:
I'm not sure about removing this actually. Should be OK to send the error, we just don't want to close the connection when this specific error is received by client/server